src/recall.cpp

```diff
@@ -426,7 +423,9 @@ static void memoryMagicSkills(uint32_t memory_spell_flags, uint32_t monster_spel
 
 // Do we know how hard they are to kill? Armor class, hit die.
 static void memoryKillDifficulty(Creature_t *creature, uint32_t monster_kills) {
-    if (!knowarmor(creature->level, monster_kills)) {
+    // the higher the level of the monster, the fewer the kills you need
+    // Original knowarmor macro inlined
+        if (monster_kills <= 304u / (4u + creature->level)) {
```

Comment:
It is needed. I tried to compile to the code without it and the result was a compiler error.